decidim-comments/config/locales/ca.yml

```diff
@@ -12,23 +12,23 @@ ca:
         manage_email_subscriptions_html: Pots deixar de rebre aquests correus electrònics canviant la configuració a %{link}.
         notifications_settings_link: la pàgina de configuració de les notificacions
         reply_created:
-          new_reply_html: Hi ha una nova resposta de <b>%{commenter}</b> al teu comentari a <b>%{commentable_link}</b>
+          new_reply_html: Hi ha una nova resposta del teu comentari de l'<b>%{commenter}</b> a <b>%{commentable_link}</b>
```

Comment:
Also, we cannot ensure the name of the commenter starts with a vowel, so we need to find another way to do this.